sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/AuthTokenContext.java

```diff
@@ -46,6 +46,12 @@
     @VisibleForTesting
     static final int TOKEN_HISTORY_LIMIT = 5;
 
+    /**
+     * The minimum length of account Id.
+     */
+    @VisibleForTesting
+    static final int ACCOUNT_ID_LENGTH = 36;
```

Comment:
I would suggest adding 'max' to the variable name, e.g. `ACCOUNT_ID_MAX_LENGTH`

Reply:
It's fixed length, not variable so always max. The code for min is just a guard.